Src/Newtonsoft.Json/Converters/IsoDateTimeConverter.cs

```diff
@@ -57,7 +57,7 @@ public DateTimeStyles DateTimeStyles
         public string DateTimeFormat
         {
             get { return _dateTimeFormat ?? string.Empty; }
-            set { _dateTimeFormat = StringUtils.NullEmptyString(value); }
+            set { _dateTimeFormat = (string.IsNullOrEmpty(value)) ? null : value; }
```

Comment:
It doesn't. I just got rid of a method that is only used once at the same time.

Wrapping boolean expressions in parenthese like they are in an if statement is an old habit of mine.